src/Middleware/RecordTransaction.php

```diff
@@ -50,6 +50,7 @@ public function handle(Request $request, Closure $next)
 
         // Start a new transaction
         $transaction = $this->startTransaction($transaction_name);
+        $this->agent->setCurrentTransaction($transaction);
```

Comment:
Are we interested in distributed transaction in jobs? If so, we will need add this to the JobCollector as well.

Reply:
We need to be aware of the impact on long running workers, but since this is in the http middleware, the current transaction actually would not be set for a job. That raises an entirely different problem to address since the job should also be able to access the current transaction for distributed tracing. I'll make a note of this and work it into the job middleware.

Reply:
Grr. I thought was replying to the next comment. That's what I get for readying comments in email first.

Yes, you are exactly correct and I'll work on addressing this.